providers/provider-protocol-reference.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Provider Protocol Reference
+category: 63d7e8bdbf7b4b0e0745e823
+slug: provider-protocol-reference
+---
+
+# Provider Protocol Reference (v2)
+
+All functions decribed here take msgpack-serialized payloads as arguments and
+return values. The key/values to use for these payloads are exactly the ones you
+can find in `fiberplane::models` crate documentation (NOTE: put the docs.rs link
+once fiberplane is open-sourced and published).
```

Comment:
It appears the audience of this are Rust developers, so I wouldn't mention MessagePack or key/value payloads in this context, given they will not be directly exposed to it. Just refer to `fp-bindgen` as our bindings generator and focus on the Rust API, I'd say.